pkgs/applications/video/recordmydesktop/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchsvn, automake, autoconf, zlib, popt, xorg, libvorbis, libtheora }:
+
+stdenv.mkDerivation rec {
+  name = "recordmydesktop-${version}";
+  version = "0.3.8-svn602";
+
+  src = fetchsvn {
```

Comment:
is there a specific reason for using svn? there is a 0.3.8.1 tar.gz available on sourceforge.